plugins/postcss-is-pseudo-class/test/browser.expect.css

```diff
@@ -0,0 +1,178 @@
+/* https://github.com/web-platform-tests/wpt/blob/master/css/selectors/is-nested.html */
```

Comment:
There are still instances of `:is` in this file and that is ok for now.
These are taken from WPT and currently pass in latest chrome.

This tells us that what we transform doesn't break things.
The remaining untransformed bits work because latest chrome natively supports `:is`.

This will be a good guide going forward to make sure that future transforms for complex selectors do not introduce issues.